### ISSUE TITLE
Play random movie

### ISSUE THREAD

**Issue by helgeim** [boxeehacks/boxeehack#138]
Possible to add "Play random movie" in the movie menu?

**Comment by Silent337** [comment]
Actually would it be possible to play just random content from selected folders in succession. A cool way to create a TV station of all your favourite content. :)

**Comment by war59312** [comment]
Yes, would be sweet. I've love a "Play Random TV Episode" button.

Of course I would need to control the poll of episodes that it pulls from.

**Comment by reeps** [comment]
This would be fantastic! Creating your own TV station would be awesome. Basically set up a playlist of random episodes of a specific show. Much like TBS would play 2 random episodes of Seinfeld, then 2 random episodes of King of Queens or something along those lines.
